lib/interfaces/exports.js

```diff
@@ -56,3 +56,5 @@ module.exports = function(suite) {
     }
   }
 };
+
+module.exports.description = 'Node.js module ("exports") style';
```

Comment:
Not a fan of this change.

Would not a better approach to have been to export a `meta` object which included a `description` property? That way we could extend it. For example, reporters could annotate their stream compatibility (e.g.,`isBrowserOnly`, `isFileCapable`).

Reply:
I don't think I understand why this is problematic.  There's no reason we can't  iterate over keys or add arbitrary stuff to `exports`?

The point of this change was to restore the descriptions.  I don't really want to get into what a nice plugin convention should look like...

Reply:
Made comment here, but was really more concerning reporters.

I have a NRFPT reporter PR where I used an object to do the same things I've explained. While I didn't need `description` in mine, but integrating with this PR would require unnecessary rewrite upon merge for whoever gets there last.

It's not that big a change now to use an object property vs. string (e.g., `meta.description` vs. `description`).